### ISSUE TITLE
Redirection from short to full URL not working

### ISSUE THREAD

**Issue by intelliant01** [rtfd/readthedocs.org#4657]
## Details

* Read the Docs project URL: https://mss-doc.readthedocs.io/
* Build URL (if applicable): 
* Read the Docs username (if applicable): maintainers cyberorg, shrenik 

## Expected Result

mss-doc.rtfd.io should have redirected to mss-doc.readthedocs.io

## Actual Result

No redirection and no response.

However, the redirection works for all translation project URLs - https://mss-doc-gu.rtfd.io and https://mss-doc-hi.rtfd.io

Is this in some way related to the transition to htttps ?

**Comment by davidfischer** [comment]
This is related to the HTTPS transition for custom domains, I believe. I think this was one of about a dozen projects that I emailed because the DNS for custom domains was misconfigured. Am I right in assuming that you previously used a custom domain mss-doc.cyberorg.info for this project? It looks like this isn't used any longer.

There was a misconfiguration in the DNS setup for mss-doc.cyberorg.info to use it with Read the Docs. I had added a workaround to Read the Docs' DNS to make this work but since that domain is now removed, that is no longer necessary. This workaround was causing issues with the short URL. I have removed this workaround and once DNS propagates, that short redirection will work again.

I still see a CNAME setup in your DNS for mss-doc.cyberorg.info. I think it is different than before, but it is still not quite correct. If you want to use the domain mss-doc.cyberorg.info on Read the Docs, make sure that the CNAME is set to "readthedocs.io". Here's our [custom domain docs for reference](https://docs.readthedocs.io/en/latest/alternate_domains.html#custom-domain-support).

    $ dig mss-doc.cyberorg.info CNAME +short
    mss-doc.readthedocs.io.    # this should be just "readthedocs.io"
